.kitchen.yml

```diff
@@ -206,6 +206,31 @@ suites:
       - recipe[hadoop::zookeeper]
       - recipe[hadoop::zookeeper_server]
     attributes: { hadoop: { distribution: 'bigtop', distribution_version: '0.8.0' } }
+  - name: bigtop100
+    excludes: debian-6.0.10, ubuntu-14.04
```

Comment:
Looks like I miss read the version to codename table.  You're right it only supports Ubuntu 14.  I'll fix the test case exclude.